src/background.js

```diff
@@ -18,6 +19,49 @@ import store from "./media-manager/store";
 
 const isDevelopment = process.env.NODE_ENV !== "production";
 
+// We need to ask macOS permission to access media devices
+async function getMediaPermission() {
+  let accessGranted = false;
+
+  accessGranted = await systemPreferences.askForMediaAccess("microphone");
+  accessGranted = await systemPreferences.askForMediaAccess("camera");
```

Comment:
Whem `microphone` is not allowed, but `camera` is allowed, then this will still return `true`, even when we have no access to the `microphone` or am I wrong?

Reply:
Ah yes, you're right. My bad!